include/taco/util/env.h

```diff
@@ -43,7 +44,7 @@ inline std::string getTmpdir() {
     // ensure that we use a taco tmpdir unique to this process.
     auto tacotmpdirtemplate = tmpdir + "taco_tmp_XXXXXX";
     char *ctacotmpdirtemplate = new char[tacotmpdirtemplate.length() + 1];
-    std::strcpy(ctacotmpdirtemplate, tacotmpdirtemplate.c_str());
+    strcpy(ctacotmpdirtemplate, tacotmpdirtemplate.c_str());
```

Comment:
I think you should be able to keep the `std::` if you include `cstring` instead of `string.h`; that's probably more C++-esque.